packages/core-mobile/app/new/common/components/QrCodeScanner.tsx

```diff
@@ -83,19 +82,26 @@ Props): React.JSX.Element | undefined => {
     checkAndroidPermission()
   }, [checkAndroidPermission])
 
-  const shouldShowCamera =
+  const cameraPermissionNotGranted =
     (permission?.granted === true && Platform.OS === 'ios') ||
     (isAndroidPermissionGranted === true && Platform.OS === 'android')
 
```

Comment:
@ruijialin-avalabs the qr code scanner was showing red text (the instructions) briefly. I fixed it in this pr.